ios/FluentUI.Demo/FluentUI.Demo/Resources/Assets.xcassets/CommandBar/delete24Filled.imageset/Contents.json

```diff
@@ -0,0 +1,15 @@
+{
+  "images": [
+    {
+      "filename": "ic_fluent_delete_24_filled.svg",
```

Comment:
why we using svg? most of the fluent uses pdf

Reply:
Overlooked the filetype. Updated to the pdf asset.